sass/etna.scss

```diff
@@ -540,10 +540,47 @@ Modified to match the existing site (for now)
 // tna-useful-info component - 'Good to know' information from event page
 
 .tna-useful-info {
```

Comment:
Could this please be split out into a separate SCSS file?

Reply:
Done in https://github.com/nationalarchives/ds-wagtail/pull/1456/commits/62a97cdb342c8975ca691627c9baff9632245383